chargebee/src/main/java/com/chargebee/android/network/CBReceiptRequestBody.kt

```diff
@@ -3,13 +3,15 @@ package com.chargebee.android.network
 internal class CBReceiptRequestBody(   val receipt: String,
                                        val productId: String,
                                        val customerId: String,
+                                       val customer: CBCustomer?,
```

Comment:
Since the class `CBReceiptRequestBody` is internal, can we use the `customer`object instead of `customerId` here?